app/views/installations/index.html.haml

```diff
@@ -1,20 +1,395 @@
 #installation.row
   .col-md-12.pagecontainer
-    .pagetitle
-      %h3 Installation and configuration
-    .panel.panel-default.codeview
-      .panel-heading Client Configuration
-      .panel-body
-        %p To start sending events, configure your client with the following DSN value:
-        .codepanel
-          %span.line https://451cf228845b497b919281b158209a59:157cbe7bf9d34b67b906dea4d3e849d0@app.feedbacker.com/12798
-        %br/
-        %p If this is a client that only needs public access (like raven-js), you can use the following public DSN:
-        .codepanel
-          %span.line https://3291842f716643f2968b11e3c86113f9@app.feedbacker.com/12798
-    .panel.panel-default.tileview
-      .panel-heading Platforms
-      .panel-body
-        %a.tile.text-center{:href => "installations/show"}
-          %img{:src => image_path("icon_javascript.png")}/
-          %span Javascript
\ No newline at end of file
+    %br
+    .bs-example{"data-example-id" => "simple-nav-justified"}
+      %ul.nav.nav-pills.nav-justified#top-tabs
+        %li{:name => 'details',:role => "presentation"}
+          %a{:href => "#client-details"}
+            %h4 Details
+        %li.active{:name => 'client configuration', :role => "presentation"}
+          %a{:href => "#client-configuration"}
+            %h4 Client Configuration
+        %li{:name => 'integrations', :role => "presentation"}
+          %a{:href => "#client-integrations"}
+            %h4 Integrations
+      %br
+      #client-integrations.tab.main-tabs
+        %br
+        %br
+        .col-lg-12
+          .panel.panel-default
+            .panel-heading Manage Integrations
+            .panel-body
+              %table.table.table-striped
+                %thead
+                  %tr
+                    %th{:style=> "width:1200px;"} Integration
```

Comment:
nu stiu la ce naiba te referi. Eu ziceam ca nu e bine sa pui width fix la nici un element si mai ales sa pui style. Parca am stabilit ca nu facem asa ceva. 

Daca pui width fix si cu style mai ales atunci cand te uiti din tableta sau telefon o sa iti faca scroll orizontal. Not good!!